site/gatsby-site/plugins/gatsby-theme-i18n-react-i18next/index.js

```diff
@@ -0,0 +1 @@
+// noop
```

Comment:
Is this needed?

Reply:
Looks like it is not, but many gatsby packages do that so 🤷

Reply:
let's keep it as it is then 👍